Add blog post for Jellyfin Android 2.5.0 release (#532)
jellyfin/jellyfin.org: blog/2023/05-14-android-v250/index.mdx

---

> title: 'Jellyfin Android v2.5.0 - Bitrate selection, player improvements and quality-of-life tweaks'

Reviewer: Title might be a bit too long, probably makes sense to the second part to the description

description: 'Enjoy a more polished Jellyfin experience on your phone'
authors: Maxr1998
date: 2023-05-14
slug: android-v2.5.0
---

> It has been a long while since the last update, but after an extended beta testing period with plenty of bug reports (thanks, by the way!) and subsequent fixes, we're excited to finally announce a new major update to Jellyfin Android.

Reviewer: ```suggestion
It has been a long while since the last update, but after an extended beta testing period with plenty of bug reports (thanks, by the way!) and subsequent fixes, we're excited to finally announce a new major update to the Jellyfin app for Android.
```

(or something similar)

> Version 2.5.0 again focused on improvements to the integrated player, and brings various new features, many bug fixes and a lot of technical maintenance and rewrites.

Reviewer: ```suggestion
Version 2.5.0 again focuses on improvements to the integrated player, and brings various new features, many bug fixes and a lot of technical maintenance and rewrites.
```

Not sure about the "again" part, maybe write it as "just like the previous version" or something.

Author: I'd like to keep the again, 2.4.0 was similar after all.

The player interface itself now has a back button to close the player and a new playback option was added to pick between hardware and software decoding, if the default hardware decoder causes issues for the currently played video.

In the client settings (which you should make sure to check out) we added the option to pick the initial orientation for video playback and now allow choosing whether you want video with SSA subtitles to be transcoded (for full styling support) or not (to direct play with limited styling). There's also a link to the operating system's subtitle customization settings, which the integrated player uses to style your subtitles. Note that the subtitle settings in the Jellyfin settings only apply to the web player due to technical reasons.

## Other Tweaks

> Thanks to @susonthapa, the app now shows a splash screen with the Jellyfin logo on start, instead of the previous, boring gray screen where you didn't see whether the app was actually doing anything. While the app is connecting, there's now a cancel button so that you can quickly jump to the connect screen to select a different server if the current one isn't available at the moment.

Reviewer: ```suggestion
Thanks to [@susonthapa](https://github.com/susonthapa), the app now shows a splash screen with the Jellyfin logo on start, instead of the previous, boring gray screen where you didn't see whether the app was actually doing anything. While the app is connecting, there's now a cancel button so that you can quickly jump to the connect screen to select a different server if the current one isn't available at the moment.
```

> The connect screen itself was reworked and rewritten in JetPack Compose, a new UI framework by Google that allows fast and easy UI development. We plan to use it for more parts of the app in the future, especially for the long-term goal of a completely native app experience.

Reviewer: ```suggestion
The connect screen itself was reworked and rewritten in Jetpack Compose, a new user interface framework by Google that allows fast and easy UI development. We plan to use it for more parts of the app in the future, especially for the long-term goal of a completely native app experience.
```

This part is a bit technical, I try to avoid those things in my own blogposts assuming the average reader doesn't understand/care.

Author: Gonna remove that part then and just mention the rewrite for a cleaner design.

We also added a monochrome icon so that Jellyfin will look great on your home screen on supported launchers on Android 13 and above.

## TODO – To mention or not to mention?

- Cronet - very technical and most probably don't care. The percentage of users that deployed QUIC is probably also quite low
- PGSSUB - a full sentence for just one subtitle codec? Idk..

> - Disabled Safe browsing - only applies to very few users and might confuse/scare others

Reviewer: Cornet: not worth mentioning imo
PGSSUB: could be added to the integrated player improvements, after the SSA part maybe? "We've also added support for PGSSUB without the need for transcoding"
Safe browsing: not worth mentioning imo

> ## Download Now

Reviewer: I normally place the download section beneath the contributors to highlight those that worked on the release. Also easy to just scroll down to the bottom of the page to find it.

<a className="margin-right--md" href="https://play.google.com/store/apps/details?id=org.jellyfin.mobile">
  <img width="153" alt="Jellyfin on Google Play" src="/images/store-icons/google-play.png" />
</a>

<a className="margin-right--md" href="https://www.amazon.com/gp/product/B081RFTTQ9">
  <img width="153" alt="Jellyfin on Amazon App Store" src="/images/store-icons/amazon.png" />
</a>

<a href="https://f-droid.org/en/packages/org.jellyfin.mobile/">
  <img width="153" alt="Jellyfin on F-Droid" src="/images/store-icons/fdroid.png" />
</a>

> Direct downloads are always available from [our repository](https://repo.jellyfin.org/releases/client/android/).

Reviewer: ```suggestion
Direct downloads are available for sideloading at [repo.jellyfin.org](https://repo.jellyfin.org/releases/client/android/)
or in the [GitHub release assets](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).
You can also join our [beta program on Google Play](https://play.google.com/apps/testing/org.jellyfin.mobile) and
help test new versions before they're released to the public.
```

## Full Release Notes

> The full (technical) release notes are available on [GitHub](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).

Reviewer: ```suggestion
The complete list of changes for this release, including all merged pull requests, can be found on
[GitHub](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).
```

Author: I'm not sure how to feel about the “including”, it *is* just the pull requests..

Reviewer: How about

```suggestion
The complete list of changes for this release with all merged pull requests can be found on
[GitHub](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).
```

Author: Better.

## Contributors

> As always, many great people contributed to this release:

Reviewer: ```suggestion
Jellyfin apps are developed by our great contributors and we couldn't do it without them. Everyone is a volunteer that
doesn't get paid by any organizations, so consider donating if you appreciate their work. A big shout-out to all
contributors that made this release possible:
```

> ### Jellyfin Team

Reviewer: ```suggestion
**Jellyfin Team**
```

> - [@nielsvanvelzen](https://github.com/nielsvanvelzen) - [Sponsor](https://github.com/sponsors/nielsvanvelzen)
> - [@Maxr1998](https://github.com/Maxr1998) - [Sponsor](https://github.com/sponsors/Maxr1998)

Reviewer: ```suggestion
- [@nielsvanvelzen](https://github.com/nielsvanvelzen) - Sponsor via [GitHub sponsors](https://github.com/sponsors/nielsvanvelzen)
- [@Maxr1998](https://github.com/Maxr1998) - Sponsor via [GitHub sponsors](https://github.com/sponsors/Maxr1998)
```

> ### Others

Reviewer: ```suggestion
**Other contributors**
```

---
title: 'Jellyfin Android 2.5.0'
description: 'Bitrate selection, player improvements and quality-of-life tweaks'
authors: Maxr1998
date: 2023-05-14

> slug: android-v2.5.0

Reviewer: ```suggestion
slug: android-v2.5.0
tags: [release, android]
```
